tests/Nelmio/Alice/support/models/PropertiesParent.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Nelmio\Alice\support\models;
+
+class PropertiesParent
```

Comment:
Sure, it just didn't make sense to use a class called PluralProperties for test that doesn't work with arrays. But as you wish, I'll change it back.